Gemfile

```diff
@@ -23,4 +23,8 @@ group(:dev) do
   gem 'guard'
   gem 'guard-rspec'
   gem 'ruby_gntp'
+  gem 'chefspec'
+  gem 'rubocop'
+  gem 'knife-spork'
+  gem 'kitchen-vagrant'
```

Comment:
i think you need to either rename the group to `:development` or else include the line:

```
gemspec :development_group => :dev
```